package-lock.json

```diff
@@ -44,7 +44,6 @@
         "convict": "6.2.4",
         "dotenv": "16.0.3",
         "fastify": "4.11.0",
-        "jose": "4.13.1",
```

Comment:
Guess it was added in development, I think it should not be removed in package-lock -- reinstall dependencies